packages/task/src/browser/task-configurations.ts

```diff
@@ -198,48 +207,46 @@ export class TaskConfigurations implements Disposable {
      * If reading a config file wasn't successful then does nothing.
      */
     protected async refreshTasks(configFileUri: string) {
-        const tasksConfigsArray = await this.readTasks(configFileUri);
-        if (tasksConfigsArray) {
+        const configuredTasksArray = await this.readTasks(configFileUri);
```

Comment:
Tasks from a config file are read twice per one call refreshTasks() method:
1. `await this.readTasks(configFileUri);` ----> `this.getTaskConfigurationsFromFile(uri);`
2. `await this.readTaskCustomizations(configFileUri);` ----> `this.getTaskConfigurationsFromFile(uri);`

Is it expected?

Reply:
You are right. having one read would be good enough. will improve.